src/Dapr.Actors/Runtime/ReminderInfo.cs

```diff
@@ -41,6 +43,8 @@ internal struct ReminderInfo
         public byte[] Data { get; private set; }
 
         public TimeSpan? Ttl { get; private set; }
+        
+        public int? Repetitions { get; private set; }
```

Comment:
Should've asked this earlier, but how do ttl and repetitions interact? Is it OK to have both?

Reply:
A very good catch, on reading the docs, I found that it is OK to have both. Adding a method overload.